invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not sure it's a good idea to have the full blown setup.py. I think a a very minimal setup.py file is better in order to allow beginners to understand what's going on and what is the bare minimal need.

We can recommend in the docs that they can use our cookiecutter module if they want a full blown setup.